pkg/test/environment.go

```diff
@@ -76,7 +76,7 @@ func NewEnvironment(ctx context.Context, options ...EnvironmentOption) *Environm
 		Environment: envtest.Environment{
 			CRDDirectoryPaths: []string{
 				project.RelativeToRoot("charts/karpenter/crds/karpenter.sh_provisioners.yaml"),
-				project.RelativeToRoot("charts/karpenter/crds/karpenter.k8s.aws_awsnodetemplate.yaml"),
+				project.RelativeToRoot("charts/karpenter/crds/karpenter.k8s.aws_awsnodetemplates.yaml"),
```

Comment:
This should just work.
```suggestion
				project.RelativeToRoot("charts/karpenter/crds"),
```